index.html

```diff
@@ -6,6 +6,392 @@
+		<section id="humanBody">
+
+			<!-- The diagram is a series of figures, with respective captions extending into tag bubbles -->
+
+			<figure>
+				<img src="assets/body.svg" alt="a gray outline of a human body">
+				<figure class="tagRight" id="comprehension">
+					<img src="assets/icon-comprehension.svg" alt="Two intertwined gears">
+					<figcaption>
+					<h3>Comprehension</h3>
+					“All of that medical jargon doesn’t make much sense to me but I don’t want to bother the doctor with my silly questions.”
+					</figcaption>
+				</figure>
+				<figure class="tagLeft" id="denial">
+					<img src="assets/icon-denial.svg" alt="A heart symbol">
+					<figcaption>
+					<h3>Denial</h3>
+					“I feel just fine. I don’t know why that doctor thinks I’m sick. Why would I take medicine that I don’t need?”
+					</figcaption>
+				</figure>
+				<figure class="tagRight" id="cost">
+					<img src="assets/icon-cost.svg" alt="A dollar sign">
+					<figcaption>
+					<h3>Cost</h3>
+					“I took my prescription to the pharmacy but they said it would cost several hundred dollars! I just couldn’t afford it.”
+					</figcaption>
+				</figure>
+				<figure class="tagLeft" id="complexity">
+					<img src="assets/icon-complexity.svg" alt="Five tally marks">
+					<figcaption>
+					<h3>Complexity</h3>
+					“Taking 15 medications every day is tough. Especially since they all have different instructions. How am I supposed to keep track of everything?”
+					</figcaption>
+				</figure>
+			</figure>
+		</section>
+	</section>
+	<!-- end why people don't adhere -->
+
+	<!-- The Question -->
+	<section class="row">
+		<span class="subTitle">THE QUESTION</span>
+		<h2 class="sectionDescription">How can technology help patients deal with these factors, so they adhere more accurately and consistently?</h2>
+		<!-- Float both the sub-article and the image left -->
+		<article>
+			<p>
+			Access was our primary concern. Smartphone adoption is growing but 35% of the U.S. population still doesn’t own one.<span>4</span> Smartphones offer many additional features but impose a barrier of entry that seemed too high. Reliance on personal computers is also problematic, especially among disadvantaged patients. 
+			</p>
+			<p>
+			We narrowed our exploration to technology that was pervasive and familiar. Nearly 9 out of 10 American adults use a phone capable of texting.<span>5</span> Furthermore, adherence studies using text reminders show promising results.<span>6</span>  Texting shows the potential to address difficult issues around the world and across verticals. In Libya, the development of the world’s first SMS voter registration system led to 1.5 million voter registrations.<span>7</span>
+			</p>
+			<p>
+			Could texts be used not only to send reminders but also to build an ongoing two-way dialogue with a trusted health partner? Would this conversation help improve medication adherence?
+			</p>
+		</article>
+
+		<!-- rotate the figure using css -->
+		<figure>
```

Comment:
It still needs `alt=""` though.  No alt text at all means a screen reader will read the filename.